build_data/stable_plugins.txt

```diff
@@ -3,6 +3,7 @@ authkey-plugin
 cas-plugin
 charts-plugin
 css-plugin
+csw-plugin
```

Comment:
Did you check this line here https://github.com/kartoza/docker-geoserver/blob/2.18.0/scripts/setup.sh#L49. The CSW plugin is installed by default that's why it's not added in the stable extensions

Reply:
Oh sorry my bad, forgot to check there !  Then no need for this PR anymore and we can just enable `cws-iso-plugin` directly.  Thanks !

Reply:
Cool and by the way why didn't you migrate to 2.19.0 since it's the new kid on the block

Reply:
Oh that's weird, last night, on DockerHub, sorted by newest, I did not see 2.19.0 !  Sure thing we'll move to 2.19.0.

Reply:
By the way, the list of default activated plugins on the README https://github.com/kartoza/docker-geoserver/blob/master/README.md#activate-plugins-on-runtime is not up-to-date.  I initially looked there so that's why I thought `csw-plugin` was missing in `stable_plugins.txt`.